web/src/css/codemirror.less

```diff
@@ -0,0 +1,5 @@
+.ReactCodeMirror {
+	border: 1px solid #ccc;
+}
+
```

Comment:
How about adding the `height: auto` part from https://codemirror.net/demo/resize.html, so that we can get rid of one of the scrollbars? :smiley: